dbms/src/Storages/KVStore/MultiRaft/PreHandlingTrace.h

```diff
@@ -31,8 +31,19 @@ struct PreHandlingTrace : MutexLockWrap
         {}
         std::atomic_bool abort_flag;
     };
+
+    class KVStore * kvstore;
```

Comment:
why we need this ptr as a member of `PreHandlingTrace`

Reply:
`updateProxyConfig` can only be called some time after the KVStore is constructed